src/common/messages.ts

```diff
@@ -57,6 +58,7 @@ export class Messages {
             Toggle: `${messagePrefix}/visualization/toggle`,
             ScanCompleted: `${messagePrefix}/visualization/scanCompleted`,
             ScrollRequested: `${messagePrefix}/visualization/scrollRequested`,
+            RescanVisualization: `${messagePrefix}/visualization/reScanVisualization`,
```

Comment:
This is the only place where we capitalize the 'S' in reScan. For consistency, maybe lower case here, too?
```suggestion
            RescanVisualization: `${messagePrefix}/visualization/rescanVisualization`,
```